core/src/main/scala/org/apache/spark/deploy/SparkHadoopUtil.scala

```diff
@@ -111,6 +110,14 @@ class SparkHadoopUtil extends Logging {
   }
 
   /**
+    * Return an appropriate (subclass) of Configuration. Creating config can initializes some Hadoop
+    * subsystems.
+    */
+  def newConfiguration(conf: SparkConf): Configuration = {
```

Comment:
I'm curious where this is used after your change... generally it's not a great idea to make a new `Configuration` since there is probably one with appropriate configuration available that should be used with your new method.